AdminWebsite/AdminWebsite/Program.cs

```diff
@@ -35,7 +37,8 @@ private static IHostBuilder CreateWebHostBuilder(string[] args)
                     });
                     webBuilder.ConfigureAppConfiguration(configBuilder =>
```

Comment:
Thought for new prs we were going to remove the duplicate provider registrations?